src/views/hoc/hoc.scss

```diff
@@ -8,123 +8,117 @@ $base-bg: $ui-white;
 }
 
 .hoc {
```

Comment:
If we're transitioning to the Trello style, then I think we should avoid the top-level nesting we were using for encapsulation before. Instead of putting everything as a descendant to `.hoc`, the component classes that are specific to the `.hoc` view should be prefixed with `.hoc-`.